modules/ROOT/pages/tutorial-setup.adoc

```diff
@@ -83,7 +83,7 @@ To make the tutorial simpler, you should rename the image that we have just down
 
 [source,bash]
 ----
-mv fedora-coreos-32.20200715.3.0-qemu.x86_64.qcow2 fedora-coreos.qcow2
+mv fedora-coreos-<version_number>-qemu.x86_64.qcow2 fedora-coreos.qcow2
```

Comment:
There are a few more cases like this in the rest of the page. I think it's easier to set a `$RELEASE` variable and use it later on. Something like:
```
coreos-installer download -p qemu -f qcow2.xz --decompress
RELEASE="32.20200715.3.0" # Change this according to the current version that has been downloaded.
```

Reply:
coreos-installer prints the full path to the downloaded file to stdout, so we could just capture that into a variable.